Sources/PostgresKit/PostgresClient+SQL.swift

```diff
@@ -1,13 +1,20 @@
+import PostgresNIO
+import Foundation
 import SQLKit
 
 extension PostgresDatabase {
-    public func sql() -> SQLDatabase {
-        _PostgresSQLDatabase(database: self)
+    public func sql(
+        encoder: PostgresDataEncoder = PostgresDataEncoder(jsonEncoder: JSONEncoder()),
```

Comment:
these can be just `encoder: PostgresDataEncoder = .init()` so that we minimize code dupe